apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss

```diff
@@ -1,84 +1,114 @@
 @import "phase1-design-system";
 
 $border-radius: 4px;
+$container-width: 309px;
 
 .curriculumCatalogCardContainer {
   display: flex;
   flex-flow: column;
   background-color: $neutral_light;
   border-radius: $border-radius;
-  height: 384px;
-  width: 309px;
+  width: $container-width;
 
   img {
     height: 140px;
     object-fit: cover;
     border-radius: $border-radius $border-radius 0 0;
   }
+}
 
-  .curriculumInfoContainer {
-    display: flex;
-    flex-flow: column;
-    height: 100%;
-    border: 1px solid $neutral_dark20;
-    border-top: 0;
-    border-radius: 0 0 $border-radius $border-radius;
-    padding: 1.25em 1em;
-
-    .overline {
-      color: $brand_primary_default;
-      margin-bottom: 0.5em;
-      font-size: 0.75em;
-      font-family: $gotham-bold;
-      text-transform: uppercase;
-      letter-spacing: 0.5px;
-    }
+.curriculumCatalogCardContainer_english {
+  height: 24em;
+}
 
-    i {
-      color: $neutral_dark80;
-    }
+.curriculumCatalogCardContainer_notEnglish {
+  height: 26em;
+}
 
-    .tagsAndTranslatabilityContainer {
-      display: flex;
-      justify-content: space-between;
+.curriculumInfoContainer {
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  border: 1px solid $neutral_dark20;
+  border-top: 0;
+  border-radius: 0 0 $border-radius $border-radius;
+  padding: 1.25em 1em;
 
-      i {
-        margin-left: auto;
-        font-size: 1.5em;
-      }
-    }
+  i {
+    color: $neutral_dark80;
+  }
 
-    h4 {
-      margin-bottom: 0.75em;
-      color: $neutral_dark;
+  button {
+    margin: 0;
+
+    &:focus {
+      outline: $brand_primary_default solid 2px;
+      outline-offset: 2px;
     }
+  }
 
-    .iconWithDescription {
-      display: flex;
-      margin: 0 0 0.5em 0;
-      align-items: center;
+  h4 {
+    margin-bottom: 0.75em;
+    color: $neutral_dark;
+  }
+}
 
-      .iconDescription {
-        font-size: 0.875em;
-        margin: 0 0 0 0.5em;
-        color: $neutral_dark;
-      }
-    }
+.tagsAndTranslatabilityContainer {
+  display: flex;
+  justify-content: space-between;
 
-    .buttonsContainer {
-      display: flex;
-      flex-wrap: wrap;
-      justify-content: space-between;
-      margin-top: auto;
-
-      button {
-        height: 40px;
-        margin: 0;
-        // TODO [MEG] Adjust to accommodate different string
-        // sizes, look at gap https://css-tricks.com/almanac/properties/g/gap/
-        width: 132px; // calculated: (totalContainerWidth - 2*padding - minimumSpaceBetweenButtons) / 2
-        font-size: 1em;
-      }
-    }
+  i {
+    margin-left: auto;
+    font-size: 1.5em;
+  }
+}
+
+.iconWithDescription {
+  display: flex;
+  margin: 0 0 0.5em 0;
+  align-items: center;
+
+  .iconWithDescriptionText {
```

Comment:
I intentionally left this nested to visually connect it with the iconWithDescription. In the [BEM world](https://getbem.com/naming/), I would've called this `icon-with-description__text` because it's an element. But with the need for camel casing in JSX for the `style.keyName` syntax, I stuck with camel case here too.

I realize this is not consistent with the rest of the flattening –– open to more discussion about this!

Sidenote: Looks like there are some [libraries](https://jahed.dev/2018/02/09/using-css-modules-with-bem/) out for using CSS modules with BEM, but we haven't normed on using BEM conventions, and combining BEM with flattening the styles could also get messy.

Reply:
> I intentionally left this nested to visually connect it with the iconWithDescription. In the [BEM world](https://getbem.com/naming/), I would've called this `icon-with-description__text` because it's an element. But with the need for camel casing in JSX for the `style.keyName` syntax, I stuck with camel case here too.

BEM looks quite interesting!  If the only thing stopping you is the camel casing, you should be able to do something like `style['icon-with-description__text']`. I suppose it is a tradeoff. I am interested in your thoughts on whether we should push further in the BEM direction.

> I realize this is not consistent with the rest of the flattening –– open to more discussion about this!

This seems fine... after discussion I'm fine with being flexible on whether styles are flattened.

> Sidenote: Looks like there are some [libraries](https://jahed.dev/2018/02/09/using-css-modules-with-bem/) out for using CSS modules with BEM, but we haven't normed on using BEM conventions, and combining BEM with flattening the styles could also get messy.

If we wanted to adopt BEM, I wouldn't let my initial opinion on nested styles get in the way. adopting BEM seems like a much bigger benefit than any of my arguments against.

Reply:
I didn't want to open the BEM can of worms tehe ... but some initial thoughts
1) Our [Styleguide](https://github.com/code-dot-org/code-dot-org/blob/a70fc266fef0226d464fbc9b06c085a0c33b3558/STYLEGUIDE.md?plain=1#L393) says to use kebab-case for styling. But we aren't really following this, e.g. the `style['icon-with-description__text']` syntax is only present in 6 files in our repo.
2) I asked about the camelCase vs kebab-case in the summer swarm ([thread](https://codedotorg.slack.com/archives/C03MZ259SUU/p1660840277341799)), and turns out that CSS Modules recommends CamelCase. Perhaps the easiest lift right now is to update the styleguide.
3) We could enable a [stylelint rule](https://stylelint.io/user-guide/rules/selector-class-pattern/) for camelCase, but if my initial stylelint attempt for [linting the ID](https://github.com/code-dot-org/code-dot-org/pull/47726) is any indication, it would be a pretty big PR and would require some manual editing, as classnames show up in JS files too.
4) There are [lots of BEM naming conventions](https://en.bem.info/methodology/naming-convention/#alternative-naming-schemes), but they all involve the `-`, which would require the `style['...']` syntax. I don't think this is blocking, but it would negate the use of the stylelint rule.
5) I do like the BEM standards, but I used it in my last company and it required a fair bit of upskilling across the team to make it work. I'm not sure the time it takes to figure it out is worth the hassle? Dave, what are the pros you're seeing?

CC @sanchitmalhotra126 and @mikeharv  if you have thoughts on this ...

Reply:
> one more thought on adopting BEM is that if we want to head in that direction, it would be super handy to have a linter to help us. it's possible these pieces could work together to accomplish this:
> 
> * https://github.com/postcss/postcss-bem-linter
> * https://stylelint.io/
> * https://www.npmjs.com/package/stylelint-selector-bem-pattern
> 
> I would love to support you in pursuing this if you are interested!

We put Stylelint in our repo this summer! See initial PR https://github.com/code-dot-org/code-dot-org/pull/47591 and current [config](https://github.com/code-dot-org/code-dot-org/blob/a70fc266fef0226d464fbc9b06c085a0c33b3558/apps/stylelint.config.js).

Nice finds with the BEM-specific linters. That negates a lot of my points above. I suppose the questions remaining would be
1) Do we want to adopt the BEM conventions?
2) If so, what do we do with existing styles?

Reply:
1. I'll defer to others on whether we want to head in the BEM direction. my input is that it'd be nice to have better style consistency in CSS, especially if it can be enforced via linting.

2. if using a linter, these would just be linter exceptions (via code comments) for existing styles which are hard to update. having said that... updating classnames can be tricky because you have to be very sure you are finding every instance of each classname you update or pages could break (styling and functionality), so there could be quite a lot of exceptions at first. 

one idea would be to pick a direction, and then add linting only for css modules, which we don't have very many of yet. then, as our use of css modules grows, our css will grow in the direction of a more consistent style.

Reply:
I like the idea of only linting modules for now and using BEM in those. I'll ask around a bit more, as I have lots of linter wishes!